src/tests/ftest/util/file_count_test_base.py

```diff
@@ -118,9 +119,14 @@ def run_file_count(self):
                     self.processes = ior_np
                     self.ppn = ior_ppn
                     if api == 'HDF5-VOL':
+                        env = {
+                            "HDF5_DAOS_OBJ_CLASS": oclass
+                            # "HDF5_DAOS_FILE_PROP": cont_props
+                        }
```

Comment:
We should keep the default env and update it with these. Otherwise D_LOG_FILE, etc. is removed
```suggestion
                        env = self.ior_cmd.env.copy()
                        env.update({
                            "HDF5_DAOS_OBJ_CLASS": oclass
                            # "HDF5_DAOS_FILE_PROP": cont_props
                        })
```

Reply:
Done. Thanks!